src/components/Home/ActivityTicker.tsx

```diff
@@ -70,24 +68,16 @@ export default function ActivityTicker() {
   const t = useFormatMessage()
   const [filterState, setFilterState] = useState<TickerFilter>(INITIAL_TICKER_FILTER_STATE)
 
-  const {
-    data: events,
-    refetch,
-    isLoading,
-  } = useQuery({
-    queryKey: ['events'],
-    queryFn: () => Governance.get().getLatestEvents(parseTickerFilter(filterState)),
-    refetchInterval: ONE_MINUTE_MS,
-    refetchIntervalInBackground: true,
-  })
+  const { events, refetch, isLoading } = useEvents({ event_type: parseTickerFilter(filterState) })
 
   const handleApply = (filters: TickerFilter) => {
     setFilterState(filters)
   }
 
   useEffect(() => {
     refetch()
-  }, [filterState, refetch])
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why did you remove refetch from the exhaustive deps?

Reply:
it was causing me multiple re-rerenders